vcfa/remove_leftovers_test.go

```diff
@@ -114,30 +134,6 @@ func removeLeftovers(govcdClient *govcd.VCDClient, verbose bool) error {
 		}
 	}
 
-	// --------------------------------------------------------------
-	// Organizations
-	// --------------------------------------------------------------
-	if govcdClient.Client.IsSysAdmin {
-		orgs, err := govcdClient.GetAllTmOrgs(nil)
-		if err != nil {
-			return fmt.Errorf("error retrieving Organizations: %s", err)
-		}
-		for _, org := range orgs {
-			toBeDeleted := shouldDeleteEntity(alsoDelete, doNotDelete, org.TmOrg.Name, "vcfa_org", 0, verbose)
-			if toBeDeleted {
-				fmt.Printf("\t REMOVING Organization %s\n", org.TmOrg.Name)
-				err = org.Disable()
-				if err != nil {
-					return fmt.Errorf("error disabling %s '%s': %s", labelVcfaOrg, org.TmOrg.Name, err)
-				}
-				err := org.Delete()
-				if err != nil {
-					return fmt.Errorf("error deleting %s '%s': %s", labelVcfaOrg, org.TmOrg.Name, err)
-				}
-			}
-		}
-	}
-
```

Comment:
If I am not fooled - looks like Orgs got removed

Reply:
Not sure what happened. Restored and moved above Region removal (added a note explaining why)

Reply:
Wait, reverted this. VDC removal is needed.